src/features/Account/ConfirmAccountContainer.tsx

```diff
@@ -36,19 +38,17 @@ class ConfirmAccountContainer extends React.Component<
     let buttonMessage;
     let link;
     if (this.state.wasConfirmed) {
-      result = 'Account Confirmed';
-      paragraphMessage =
-        'Your account was successfully created! Click continue to start your application, view your teams or manage your account.';
-      buttonMessage = 'Continue';
-      link = '/';
+      result = DashboardText.ConfirmAccount;
```

Comment:
Check the design doc and look at HCK-92. We have different text for confirming account now. Status would be `verified` or `unverified` email.